cdw2146.md

```diff
@@ -0,0 +1,28 @@
+# Open Source Development
+
+Name: Christopher Webb
+
+UNI: cdw2146
+
+GitHub: [link](https://github.com/chriswebb09)
+
+GitHub README: [link](https://github.com/chriswebb09/chriswebb09/blob/main/README.md)
+
+# Project Proposal
+* Project Idea: Legit(ish) AR flight sim in iOS 
+* Project Idea: Integrate Apple Watch health data into AR navigation to create exercise app for AR
+* Project Idea: Browser plugin to run content of webpage through AI detection software APIs
+* Project Idea: Clean user interface that help formulate useful AI prompts 
+* Project Idea: HackerNews clone or library, that makes sorting and searching more intuitive 
+* Project Idea: Flocking behavior library for ARKit Apple 
+* Project Idea: Site to learn binary and hex interactively
+
+JS / Python / Swift / TypeScript
+
+# Project Contributions
+

```

Comment:
add link to project markdowns

Reply:
I reverted commits so you should be able to create the conflict as planned.